src/com/tapia/foss/frames/Visualizer.java

```diff
@@ -261,6 +273,21 @@ private void display()
 
     }
     
+    /*
+     * Tries parsing the input string
+     *
+     * Returns false if the parsing fails and displays a GUI window to notify the user.
+     */
+    private boolean tryParsing(String input) {
```

Comment:
Definitely a more descriptive name.  I can pull out the JOptionPane notification from this method to the code that calls it for clarity.